back-end/API/app/repositories/user_repo.py

```
@@ -0,0 +1,65 @@
+    def update_users_enterprise_id(self, original_enterprise_id, new_enterprise_id) -> list[UserReadDTO]:
+        users = User.query.filter_by(enterpriseId=original_enterprise_id).all()
+        for user in users:
+            user.enterpriseId = new_enterprise_id
+        db.session.commit()
+        return [UserReadDTO.model_validate(user) for user in users]
+
+    def get_all(self) -> list[UserReadDTO]:
+        users = User.query.options(db.joinedload(User.enterprise)).all()
+        users_sorted = sorted(users, key=lambda e: locale.strxfrm(e.name))
+        dtos = [UserReadDTO.model_validate(e) for e in users_sorted]
+        return dtos
```

Ruff (0.15.13)

[warning] Unnecessary assignment to `dtos` before `return` statement

Remove unnecessary assignment

(RET504)